src/loader.rs

```diff
@@ -347,8 +363,7 @@ impl<'b, 'w> Loader<'b, 'w> {
         } else {
             format!("FbxMesh{}", mesh_obj.object_id().raw())
         };
-
-        trace!("Loading mesh: {label}");
+        info!("Loading FBX mesh: {label}");
```

Comment:
```suggestion
        debug!("Loading FBX mesh: {label}");
```
This is too verbose, info log level is visible by default. This only serves to check if the function ran, and the message is printed a lot. At most, I'd put it to debug level.